src/panel_gwalker/_gwalker.py

```diff
@@ -84,63 +82,45 @@ class GraphicWalker(ReactComponent):
         Servable `GraphicWalker` object that creates a UI for visual exploration of the input DataFrame.
     """
 
-    object = param.DataFrame(doc="""The data to explore.""")
-    fields = param.List(doc="""Optional fields, i.e. columns, specification.""")
-    appearance = param.Selector(
-        default=None,
-        objects=[None, "media", "dark", "light"],
-        doc="""Dark mode preference: 'media', 'dark', 'light' or None.
-        If None the the appearance is derived from pn.config.theme""",
+    object: pd.DataFrame = param.DataFrame(
+        doc="""The data to explore.
+        Please note that if you update the `object`, then the existing charts will not be deleted."""
     )
-
-    config = param.Dict(
-        doc="""Optional extra Graphic Walker configuration. See the
+    fields: list = param.List(doc="""Optional fields, i.e. columns, specification.""")
+    appearance: Literal["media", "dark", "light"] = param.Selector(
+        default="light",
+        objects=["light", "dark", "media"],
+        doc="""Dark mode preference: 'light', 'dark', 'media'.
+        If not provided the appearance is derived from pn.config.theme.""",
+    )
+    # This one is added to better explain that currently only 'client' mode is supported
+    # but we envision supporting 'server' mode one day
+    computation: Literal["client"] = param.Selector(
```

Comment:
If there's only two modes, isn't a boolean parameter better?

Reply:
It would require a rename to `server`, `server_computation`, `kernel` or `kernel_computation` to make it clearer what it means. Pygwalker uses `kernel_computation`.

I would then be easier to set

```python
GraphicWalker(..., server=True) # Or
GraphicWalker(..., server_computation=True)
GraphicWalker(..., kernel=True) # Or
GraphicWalker(..., kernel_computation=True)
```

We would then need a more specific label if the checkbox is included in the app: "Server Computation".

It could work.

The argument against is that its not very clear what the alternative is. I.e. "Client Computation". Selecting between computation="client" or computation="server" in a dropdown or RadioButtonGroup is very clear.

What would you prefer @philippjfr ?

Reply:
I will change it to `server_computation`.

Reply:
renamed to `server_computation`.